src/grid/gridColumn.js

```diff
@@ -6,10 +6,9 @@ class GridColumnComponent extends React.Component {
         return !isEqual(this.props.children, nextProps.children);
     }
     render() {
-        const flexSize = this.props.flexGrow || '1';
-        const style = { flex: `${flexSize} 0 0px` };
+        const style = { flex: '1 0 0px' };
```

Comment:
I'm not following. didn't that ability already exist with _style={{ ...this.props.style, ...style }} ?_ all I see being changed is that we're forcing the 1 into 'flex' and removing the flexGrow prop